Intermittent android build failure during fontconfig

```
make[3]: Entering directory '/repo/target/android/i686-linux-android/release/build/servo-fontconfig-sys-51d7b83465512d8a/out/src'
  GEN      fcalias.h
  GEN      stamp-fcstdint
  GEN      fcftalias.h
  GEN      fcobjshash.gperf
  GEN      fcobjshash.h
Makefile:882: recipe for target 'fcobjshash.h' failed
config.status: executing src/fcstdint.h commands
config.status: creating src/fcstdint.h : _FONTCONFIG_SRC_FCSTDINT_H
config.status: src/fcstdint.h is unchanged
make[3]: Leaving directory '/repo/target/android/i686-linux-android/release/build/servo-fontconfig-sys-51d7b83465512d8a/out/src'
Makefile:562: recipe for target 'all-recursive' failed
make[2]: Leaving directory '/repo/target/android/i686-linux-android/release/build/servo-fontconfig-sys-51d7b83465512d8a/out'
Makefile:445: recipe for target 'all' failed
make[1]: Leaving directory '/repo/target/android/i686-linux-android/release/build/servo-fontconfig-sys-51d7b83465512d8a/out'
makefile.cargo:61: recipe for target '/repo/target/android/i686-linux-android/release/build/servo-fontconfig-sys-51d7b83465512d8a/out/libfontconfig.a' failed

--- stderr
configure: WARNING: using cross tools not prefixed with host triplet
make[1]: warning: -jN forced in submake: disabling jobserver mode.
/root/.cargo/registry/src/github.com-1ecc6299db9ec823/servo-fontconfig-sys-4.0.7/missing: line 81: gperf: command not found
WARNING: 'gperf' is missing on your system.
         You might have modified some files without having the proper
         tools for further handling them.  Check the 'README' file, it
         often tells you about the needed prerequisites for installing
         this package.  You may also peek at any GNU archive site, in
         case some other package contains this missing 'gperf' program.
make[3]: *** [fcobjshash.h] Error 1
make[3]: *** Waiting for unfinished jobs....
make[2]: *** [all-recursive] Error 1
make[1]: *** [all] Error 2
make: *** [/repo/target/android/i686-linux-android/release/build/servo-fontconfig-sys-51d7b83465512d8a/out/libfontconfig.a] Error 2
thread 'main' panicked at 'assertion failed: Command::new("make").env("MAKEFLAGS",
                         env::var("CARGO_MAKEFLAGS").unwrap_or_default()).args(&["-R",
                                                                                 "-f",
                                                                                 "makefile.cargo"]).status().unwrap().success()', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/servo-fontconfig-sys-4.0.7/build.rs:20:5
```

--- a/etc/taskcluster/docker/build.dockerfile
+++ b/etc/taskcluster/docker/build.dockerfile
@@ -11,6 +11,9 @@ RUN \
         make \
         cmake \
         #
+        # Fontconfig
+        gperf \
+        #
         # ANGLE
         xorg-dev \
         #

